versioned/persist/dynamodb/src/test/java/org/projectnessie/versioned/persist/dynamodb/ITVersionStoreDynamo.java

```diff
@@ -16,7 +16,12 @@
 package org.projectnessie.versioned.persist.dynamodb;
 
 import org.projectnessie.versioned.persist.tests.AbstractDatabaseAdapterVersionStoreTest;
+import org.projectnessie.versioned.persist.tests.extension.NessieDbAdapterConfigItem;
 import org.projectnessie.versioned.persist.tests.extension.NessieExternalDatabase;
 
 @NessieExternalDatabase(LocalDynamoTestConnectionProviderSource.class)
+@NessieDbAdapterConfigItem(name = "retry.initial.sleep.millis.lower", value = "25")
```

Comment:
nit: can these settings be inherited from a superclass?

Reply:
Nope. Wanna leave the defaults for in-memory + rocks for example.

Reply:
Ah - can be moved to an interface